### PR TITLE
Wyatt Wicks Pull Request Individual Challenge 1

### PR DESCRIPTION
**Pull request by Wyattwicks**

### CODE REVIEW

File: lib/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    closing_time_int = opening_time.to_i + hours_open
+    closing_time = "#{closing_time_int.to_s}:00"
+  end
+
+  def add_dish(dish)
+    @dishes << dish
+  end
+
+  def open_for_lunch?
+    return true if opening_time.to_i <= 12
```
Comment (robertguturing) [inline lib/restaurant.rb:22]: almost, the readme says a place is open for lunch if it is before 12, not equal, so we need to get rid of the `=` operator here

Comment (robertguturing) [reply lib/restaurant.rb:22]: Also we can turn this into a one line method by doing
`opening_time.to_i < 12` as this always evaluates to true for false.

File: lib/restaurant.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    return false if opening_time.to_i > 12
+  end
+
+  def menu_dish_names
+    menu_dish_names = []
+    @dishes.each do |dish|
+      menu_dish_names << dish.upcase
+    end
+    return menu_dish_names
```
Comment (robertguturing) [inline lib/restaurant.rb:31]: the `return` statement is redundant here as the last line of a method in ruby is the value that gets returned

File: lib/restaurant.rb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  def announce_closing_time(hours_open)
+    new_closing_time = opening_time.to_i + hours_open
+    if new_closing_time < 12
+      return "#{name} will be closing at #{new_closing_time}:00AM"
```
Comment (robertguturing) [inline lib/restaurant.rb:37]: Same comment as https://github.com/turingschool-examples/i-miss-eating-out/pull/6/files#r569848072

**Review by robertguturing — COMMENTED** [review]
Score : 3

Overall this is pretty good, some comments about following ruby convention more closely. Keep up the hard work!

File: lib/restaurant.rb (hunk 1 above)
Comment (Wyattwicks) [reply lib/restaurant.rb:22]: Ah so close, thank you Bob!

**Pull request closed without merging**